### ISSUE TITLE
Fails on watch?

### ISSUE THREAD

**Issue by OutThisLife** [koute/parcel-plugin-cargo-web#8]
Building works fine, as does initiating the server.

```
Uncaught (in promise) LinkError: Import #5 module="env" function="__extjs_cf8a18ea283e81241996f7ce9c0ecdd79d01cb70" error: function import requires a callable
```

But using parcel's watch feature causes this. Command is: `parcel static/index.html --no-cache -p 3000`

Is there a fix I'm missing? Happens w/ any usage stdweb afaik:

```rust
#![feature(proc_macro)]

#[macro_use]
extern crate stdweb;

use stdweb::js_export;

#[js_export]
fn barney_vs_tellatubbies() {
  js! {
    console.log("test")
  }
}

```

```index.js
import test from './../client/Cargo.toml'

try {
  console.log(test)
} catch (e) {
  console.error(e)
}
```

```index.html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
</head>
<body>
  <script src="./index.js" async defer></script>
</body>
</html>
```

**Comment by koute** [comment]
Thanks for the report!

This should be fixed in `cargo-web` `0.6.14` which was released 3 days ago. (At least I can't reproduce it.)

**Comment by OutThisLife** [comment]
Awesome :)
